docs/src/main/sphinx/connector/pinot.rst

```diff
@@ -122,3 +122,22 @@ SQL support
 The connector provides :ref:`globally available <sql-globally-available>` and
 :ref:`read operation <sql-read-operations>` statements to access data and
 metadata in Pinot.
+
+.. _pinot-pushdown:
+
+Pushdown
+--------
+
+The connector supports :ref:`aggregate pushdown <aggregation-pushdown>` for the
+following functions:
+
+* :func:`avg`
+* :func:`approx_distinct`
+* :func:`count` and ``count(distinct)``
```

Comment:
Only `count(*)` pushdown and `count(distinct)` are supported.

`count(column)` is not pushed down. See the comment at https://github.com/trinodb/trino/issues/9029#issuecomment-908270876 for reasoning and details.

Reply:
Reworded accordingly